packages/module-auto-icons/build.config.ts

```diff
@@ -0,0 +1,5 @@
+import { defineBuildConfig } from 'unbuild';
+
+export default defineBuildConfig({
+  externals: ['fs-extra'],
```

Comment:
Isn't unbuild already marking `fs-extra` as external? It should see it in the `dependencies` and know to not bundle it.

Reply:
Yeah, tested and it's the same output without this file, I removed it in my commit.